src/OutGridView.Gui/Services/FilterBuilder.cs

```diff
@@ -57,7 +57,7 @@ static class FilterBuilder
                 return dataList.Values.Any(data =>
                     {
                         //Quick Search is NOT case-sensitive
-                        return data != null && data.Value.ToLowerInvariant().Contains(t.ToLowerInvariant());
+                        return data.Value != null && data.Value.DisplayValue.ToLowerInvariant().Contains(t.ToLowerInvariant());
```

Comment:
```suggestion
                        return data.Value?.DisplayValue.ToLowerInvariant().Contains(t.ToLowerInvariant());
```

I _think_ this is correct.